client/verta/verta/_tracking/experimentrun.py

```diff
@@ -35,12 +35,14 @@
     importer,
 )
 
-from .._dataset_versioning import (
+from .. import _repository
+from .._repository import commit as commit_module
+# unless _repository.blob is disentangled out of _repository, importing from
+# dataset must occur after importing from _repository
+from ..dataset.entities import (
```

Comment:
Potential followup is to have a submodule to house the versioning blobs
https://vertaai.slack.com/archives/CS7UJ399B/p1621453547015500?thread_ts=1621367055.002300&cid=CS7UJ399B